Draft.md

```diff
@@ -196,28 +196,26 @@ Die folgenden Fragen richten sich auf technische Merkmale, die die Benutzbarkeit
 
 #### Installation
 
-* **Wie funktioniert die Installation und wo wird die Software vorgehalten?** Gibt es ein Installationsskript oder ist die Software nur als Quellcode verfügbar, der zunächst selbst kompiliert werden muss? Installationsskripte ermöglichen einen viel breiteren Anwenderkreis, das Kompilieren erlaubt geübten Nutzern die Installation u.U. auf verschiedenen Endgeräten. Passt die gewählte Lösung zum Nutzerkreis? Welche konkreten Probleme und Schwachstellen hat der Installationsprozess?
+* **Wie funktioniert die Installation und wo wird die Software vorgehalten?** Umso einfacher die Installation einer Software, desto größer ist ihr potentieller Nutzerkreis. Die Vielartigkeit von Computersystemen macht es aber aus Entwicklersicht oft schwer, eine einfache Installation zu für alle Nutzer zu gewährleisten. Für die Rezension kann geprüft werden, ob ein Installationsskript oder sogar -wizard bereitsteht, um die Installation zu vereinfachen. Gegebenenfalls ist die Software nur als Quellcode verfügbar, der zunächst selbst kompiliert werden muss. Installationsskripte ermöglichen einen viel breiteren Anwenderkreis, das Kompilieren erlaubt geübten Nutzern die Installation u.U. auf verschiedenen Endgeräten. Verschiedene Nutzergruppen bevorzugen unterschiedliche Installationsformen. Passt die gewählte Lösung zum Nutzerkreis? Wenn der Rezensent die Software selbst installiert, wird er oft auf konkreten Probleme und Schwachstellen im Installationsprozess stoßen, die durchaus Teil der Rezension sein können.
 
```

Comment:
Das ist irgendwie in sich redundant. Macht aber nichts, noch einmal bügeln und es sitzt und hat die richtige Stärke :)

Reply:
Ggf. einfach weiter bügeln :man_shrugging: